static/app/views/dashboards/widgetCard/chart.tsx

```diff
@@ -527,19 +532,28 @@ class WidgetCardChart extends Component<WidgetCardChartProps> {
                       autoHeightResize={shouldResize ?? true}
                       noPadding={noPadding}
                     >
-                      {getDynamicText({
-                        value: this.chartComponent({
-                          ...zoomRenderProps,
-                          ...chartOptions,
-                          // Override default datazoom behaviour for updating Global Selection Header
-                          ...(onZoom ? {onDataZoom: onZoom} : {}),
-                          legend,
-                          series: [...series, ...(modifiedReleaseSeriesResults ?? [])],
-                          onLegendSelectChanged,
-                          forwardedRef,
-                        }),
-                        fixed: <Placeholder height="200px" testId="skeleton-ui" />,
-                      })}
+                      <div style={{flex: 1}}>
```

Comment:
nit: Do we commonly use inline styles? Might be worth using styled component for this since it looks like we also use another div with flex 1 somewhere further below.